config/config.go

```diff
@@ -525,6 +528,21 @@ func (r *Route) UnmarshalYAML(unmarshal func(interface{}) error) error {
 			return fmt.Errorf("invalid label name %q", k)
 		}
 	}
+	for _, l := range r.GroupByStr {
+		if l == "..." {
+			r.GroupByAll = true
+		} else {
+			labelName := model.LabelName(l)
+			if !labelName.IsValid() {
```

Comment:
It is here to preserve original behaviour.
 Since '...' is not valid label, I had to remove requirement of validation from it, but we still need validation for all other cases.

Reply:
ack!